src/main/java/frc/robot/Constants.java

```diff
@@ -38,6 +38,9 @@ public class Constants {
     public static final Transform3d FRONT_RIGHT_CAMERA_POSE =
             new Transform3d(0, -0.293, 0.55, new Rotation3d(0, Math.toRadians(25.0), 0));
 
+    public static final Transform3d SPEAKER_TARGET_POSE = // TODO: check real value
```

Comment:
This is used only in `ShootState`, so it should be in that package's constants file. Generally, constants should be in the most specific package where it's used.

Reply:
This has been removed in 60aa413